Src/EB/AMReX_EB2_Level.H

```diff
@@ -290,6 +298,11 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
     m_bndryarea.define(m_grids, m_dmap, 1, ng, mf_info);
     m_bndrycent.define(m_grids, m_dmap, AMREX_SPACEDIM, ng, mf_info);
     m_bndrynorm.define(m_grids, m_dmap, AMREX_SPACEDIM, ng, mf_info);
+
+    multicut_fcx.define(m_grids, m_dmap, 1, ng, mf_info); 
+    multicut_fcy.define(m_grids, m_dmap, 1, ng, mf_info); 
+    multicut_fcz.define(m_grids, m_dmap, 1, ng, mf_info); 
```

Comment:
These should be defined similar to `m_areafrac` below, so that they have face data rather than cell-centered data